components/tikv_kv/src/metrics.rs

```diff
@@ -3,6 +3,14 @@
 use prometheus_static_metric::*;
 
 make_auto_flush_static_metric! {
+
+    pub label_enum GcKeyMode {
+        // The enum 'txn' contain tidb and txnkv scenarios statistics.
```

Comment:
// The enum 'txn' contains both TiDB and TxnKV scenarios statistics,
        // as they have the same storage format, and use the same GC procedures.